### PR TITLE
Restore indymedia ruleset, with broken hosts removed

### PR DESCRIPTION
**Pull request by mfb**
- Remove broken hosts which caused the ruleset to be disabled
- Keep some of the hosts which are working once again.

### CODE REVIEW

File: src/chrome/content/rules/Indymedia.xml [hunk 1]
```diff
@@ -46,6 +32,8 @@ Fetch error: http://www.indymedia.org/ => https://www.indymedia.org/: (7, 'Faile
 		- liveradio ³
 		- mexico ²
 		- miami ²
+		- nantes ᵈ
+		- www.nantes ᵈ
```
Comment (Bisaloo) [inline src/chrome/content/rules/Indymedia.xml]: Works for me

File: src/chrome/content/rules/Indymedia.xml [hunk 2]
```diff
@@ -140,14 +131,12 @@ Fetch error: http://www.indymedia.org/ => https://www.indymedia.org/: (7, 'Faile
...
+<ruleset name="Indymedia.org (partial)">
 
 	<!--	Direct rewrites:
 				-->
 	<target host="indymedia.org" />
 	<target host="athens.indymedia.org" />
-	<target host="belarus.indymedia.org" />
-	<target host="www.belarus.indymedia.org" />
 	<target host="belgium.indymedia.org" />
 	<target host="*.belgium.indymedia.org" />
```
Comment (J0WI) [inline src/chrome/content/rules/Indymedia.xml]: Could you please also add some test urls for the wildcard targets? We require at least three for each wildcard.

File: src/chrome/content/rules/Indymedia.xml (hunk 1 above)
Comment (J0WI) [inline src/chrome/content/rules/Indymedia.xml]: For me too.

File: src/chrome/content/rules/Indymedia.xml (hunk 2 above)
Comment (mfb) [inline src/chrome/content/rules/Indymedia.xml]: Can you give me an example of what you're talking about.  How would I know what URLs to test?

Comment (J0WI) [inline src/chrome/content/rules/Indymedia.xml]: See:
* https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#example-ruleset-after-style-guidelines-are-applied-with-test-urls
* https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#enumerating-subdomains

Comment (mfb) [inline src/chrome/content/rules/Indymedia.xml]: I searched google for site:belgium.indymedia.org and didn't find any sites at *.belgium.indymedia.org

I am guessing this wildcard is for private sites that I don't know the URL of e.g. dev.belgium.indymedia.org or something like that?

Comment (mfb) [inline src/chrome/content/rules/Indymedia.xml]: likewise `python sublist3r.py -d belgium.indymedia.org -e Baidu,Yahoo,Google,Bing,Ask,Netcraft,Virustotal,SSL` returns only www.belgium.indymedia.org

Comment (J0WI) [inline src/chrome/content/rules/Indymedia.xml]: Therefore you should replace the wildcard with `www.belgium.indymedia.org`

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**